readme.md

```diff
@@ -1,9 +1,224 @@
-### db.rna16s project
+# 16S RNA Reference Database
 
-<!-- [![](https://travis-ci.org/era7bio/db.rna16s.svg?branch=master)](https://travis-ci.org/era7bio/db.rna16s) -->
-<!-- [![](https://img.shields.io/codacy/???.svg)](https://www.codacy.com/app/era7/db.rna16s) -->
-[![](http://github-release-version.herokuapp.com/github/era7bio/db.rna16s/release.svg)](https://github.com/era7bio/db.rna16s/releases/latest)
+[![](https://travis-ci.org/ohnosequences/db.rna16s.svg?branch=master)](https://travis-ci.org/ohnosequences/db.rna16s)
+[![](https://img.shields.io/codacy/62caae6ae58f48dca6633f2f88ed8898.svg)](https://www.codacy.com/app/era7/db.rna16s)
+[![](http://github-release-version.herokuapp.com/github/ohnosequences/db.rna16s/release.svg)](https://github.com/ohnosequences/db.rna16s/releases/latest)
 [![](https://img.shields.io/badge/license-AGPLv3-blue.svg)](https://tldrlegal.com/license/gnu-affero-general-public-license-v3-%28agpl-3.0%29)
-[![](https://img.shields.io/badge/contact-gitter_chat-dd1054.svg)](https://gitter.im/era7bio/db.rna16s)
+[![](https://img.shields.io/badge/contact-gitter_chat-dd1054.svg)](https://gitter.im/ohnosequences/db.rna16s)
 
-Project description here.
+This is a reference database of 16S sequences based on the data from [RNAcentral release 5](http://blog.rnacentral.org/2016/03/rnacentral-release-5.html).
+
+We have developed various criteria and tuned them during long testing and manual review process to filter over *9 million sequences* of RNAcentral and retrieve only around **262 000** most representative and informative sequences. The source code in this repository allows us to make generation process of this database easily reproducible for any new release of RNAcentral.
+
+
+## Filtering routine overview
+
+Here goes an overview of the general filtering routine. For more details, check the code [documentation](docs/src/main/scala/). Filtering is split in several steps:
+
+- `filter1`: filters out unrelated and uninformative *sequences*
+- `filter2`: filters out uninformative *assignments*
+- `filter3`: filters out *wrong* assignments
+
+Some more details about each step.
+
+### `filter1`
+
+This is an important filter, because out of the huge RNAcentral database it chooses only 16S sequences and filters out those that are badly classified or are not informative.
+To pass this filter sequences have to satisfy the following predicate:
+
+- the sequence ID is **not** in `blacklistedRNACentralIDs`
+- the corresponding sequence is longer than `minimum16SLength`
+- for each taxonomy association separately:
+  + annotated as an `rRNA`
+  + **not** one of those in `uninformativeTaxIDs`
+  + a descendant of either *Archaea* or *Bacteria*
+  + **not** a descendant of an *environmental* or *unclassified* taxon
+
```

Comment:
This is useful as an overview of the predicate (which could be useful for the paper, I think). I think that referring to the code in this case is not convenient. But if you want, I can remove it from here and leave only in the code.